src/lib/templateengine.js

```diff
@@ -274,33 +276,20 @@ function TemplateEngine( undefined ) {
   
   this.enableAddressQueue = $.getUrlVar('enableQueue') ? true : false;
   
-  this.backend = 'cgi-bin'; // default path to backend
+  this.backend = 'default';
+  this.backendUrl = null;
```

Comment:
`null` is quite uncommon in JS. `undefined` is the usual way to say it's not set. (This does not mean `null` is wrong. But uncommon might come unexpected)